support/android/apk/servoview/build.gradle

```diff
@@ -259,3 +259,39 @@ class ServoDependency {
     public String fileName;
     public String folderFilter;
 }
+
+apply plugin: 'maven'
+import org.gradle.api.internal.artifacts.publish.DefaultPublishArtifact
+
+uploadArchives {
+  doFirst {
+      for ( arch in ["arm", "armv7", "arm64", "x86"] ) {
```

Comment:
We can drop normal "arm", it's already deprecated. There aren't Android 4+ devices which support armeabi but not armeabi-v7a, so we can safely drop armeabi and save some compilation work & upload bandwidth. We can remove armeabi also from Servo cloud compilation tests if it's still there.

Reply:
Other than that code looks good ;)